pom.xml

```diff
@@ -127,7 +81,7 @@
     <dependency>
       <groupId>org.jenkins-ci</groupId>
       <artifactId>annotation-indexer</artifactId>
-      <version>1.11</version>
+      <version>1.12</version>
```

Comment:
Should really be picked up from `jenkins-core` instead. I guess it being an `optional` dependency is a problem. Cue [JENKINS-48578](https://issues.jenkins-ci.org/browse/JENKINS-48578).

Reply:
Without that dependency, I receive a require upper bounds dependencies error:

```
[WARNING] Rule 6: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for org.jenkins-ci:annotation-indexer:1.4 paths to dependency are:
+-org.jenkins-ci.plugins:git:4.0.0-SNAPSHOT
  +-com.infradna.tool:bridge-method-annotation:1.17
    +-org.jenkins-ci:annotation-indexer:1.4
and
+-org.jenkins-ci.plugins:git:4.0.0-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-core:2.60.3
    +-org.jenkins-ci:annotation-indexer:1.11
and
+-org.jenkins-ci.plugins:git:4.0.0-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-core:2.60.3
    +-org.jenkins-ci:bytecode-compatibility-transformer:1.8
      +-org.jenkins-ci:annotation-indexer:1.4
and
+-org.jenkins-ci.plugins:git:4.0.0-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-core:2.60.3
    +-org.kohsuke:access-modifier-annotation:1.12
      +-org.jenkins-ci:annotation-indexer:1.4
]
```

I assume that I'm "stuck" with that for now, and can revisit it when JENKINS-48578 provides a BOM for jenkins core.